packages/flutter_tools/templates/app_shared/linux.tmpl/CMakeLists.txt.tmpl

```diff
@@ -127,6 +127,11 @@ foreach(bundled_library ${PLUGIN_BUNDLED_LIBRARIES})
     COMPONENT Runtime)
 endforeach(bundled_library)
 
+set(NATIVE_ASSETS_DIR "${PROJECT_BUILD_DIR}native_assets/linux/")
```

Comment:
Please comment what this is doing in the file. We haven't fully backfilled comments, but this is a user-owned-once-instantiated file so it's especially important that it be well explained.